test/test.js

```diff
@@ -1,19 +1,25 @@
 'use strict';
 
 import test from 'ava';
+import fs from 'fs';
+import path from 'path';
 import isPlainObj from 'is-plain-obj';
 import tempWrite from 'temp-write';
 import eslint from 'eslint';
 import es5conf from '../';
 import es6conf from '../esnext';
 
-function runEslint(file, conf) {
+function runEslint(str, conf) {
   const linter = new eslint.CLIEngine({
     useEslintrc: false,
     configFile: tempWrite.sync(JSON.stringify(conf))
   });
 
-  return linter.executeOnFiles([file]).results[0].messages;
+  return linter.executeOnText(str).results[0].messages;
+}
+
+function getFileAsString (pathName) {
+  return fs.readFileSync(path.join(__dirname, pathName), 'utf-8');
```

Comment:
`readFileSync` will throw an `Error` if the file isn't found, but I guess it's pointless to handle it since this is a part of our dev tools. 🤷‍♂️